pkg/config/sync.go

```diff
@@ -314,7 +373,7 @@ func GetCurrentConfigString(currConfig *controller.OperatorConfiguration) string
 	}
 	workspace := currConfig.Workspace
 	if workspace != nil {
-		// Don't include PodSecurityContext for now as it's less easy to compare
+		// Don't include PodSecurityContext or ContainerSecurityContext for now as it's less easy to compare
```

Comment:
Isn't this being done with the call to `reflect.DeepEqual`? Or did you mean, don't explicitly state which part of the PodSecurityContext/ContainerSecurityContext was modified and instead state that it was set by the user.